.babelrc

```diff
@@ -1,13 +1,73 @@
 {
-  "presets": ["react", "env", "stage-0"],
+  "presets": [
+    "@babel/preset-react",
+    "@babel/preset-env"
+  ],
   "env": {
     "production": {
       "plugins": [
         "transform-react-remove-prop-types",
-        "transform-react-constant-elements",
-        "transform-react-inline-elements"
+        "@babel/plugin-transform-react-constant-elements",
+        "@babel/plugin-transform-react-inline-elements",
```

Comment:
The [`@babel/plugin-transform-react-inline-elements` documentation](https://babeljs.io/docs/en/babel-plugin-transform-react-inline-elements/) is very cryptic but seems to say that for legacy browser support we should also be using the `useBuiltIns` option of `@babel/preset-env`?

> When used alongside @babel/plugin-transform-runtime, polyfills (by default including Symbol) are specifically scoped to not pollute the global scope. This breaks usage with React, as it won't have access to that polyfill and will cause your application to fail in legacy browsers.
> 
> Even if ['@babel/plugin-transform-runtime', { helpers: true, polyfill: false }] is specified, it might still break, since helpers come precompiled.
> 
> In this case, we recommend importing/requiring @babel/polyfill in the entry point of your application and using @babel/preset-env with the useBuiltIns option to only include the polyfills your targets need. Alternatively, you can also import/require core-js/modules/es6.symbol by itself.

Reply:
upon a slow reading of the documentation, i think it's saying that it will break in legacy browsers only if the project is also using `@babel/plugin-transform-runtime`?

Reply:
You're right. That seems to be it. And we're not using `@babel/plugin-transform-runtime` so we're all good.